types/hooks/screens/useSignedChannelListScreenHook.types.ts

```diff
@@ -1,11 +1,12 @@
 import {ChannelList} from '../../database/schema/ChannelList.types';
+import {ContactScreenPayload} from './useChatUtilsHook.types';
 
 interface UseSignedChannelListScreenHook {
   channels: ChannelList[];
   goToChatScreen: (channel: ChannelList) => void;
   goToPostDetailScreen: (channel: ChannelList) => void;
   goToCommunityScreen: (channel: ChannelList) => void;
-  goToContactScreen: () => void;
+  goToContactScreen: ({from}: ContactScreenPayload) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `UseSignedChannelListScreenHook` interface are good. The addition of a parameter to the `goToContactScreen` function improves its flexibility by allowing it to accept a payload object. This change will allow for more dynamic behavior based on the `from` property of the `ContactScreenPayload`. However, please ensure that all usages of this function have been updated to pass the required argument.
```diff
-  goToContactScreen: () => void;
+  goToContactScreen: ({from}: ContactScreenPayload) => void;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->